stf.test/test/saros/stf/test/consistency/CreateSameFileAtOnceTest.java

```diff
@@ -29,6 +31,7 @@ public void restoreNetwork() throws Exception {
     CARL.controlBot().getNetworkManipulator().unblockOutgoingSessionPackets();
   }
 
+  @Category(FailingTests.class)
```

Comment:
This is a regression. You can look at the Jenkins history. This test case has always passed successfully.